### PR TITLE
Remove default ordering scopes

### LINKED ISSUE #485: Get rid of ordering in default scopes
Turns out it's a bad idea to use default scopes for ordering: http://pragdave.blogs.pragprog.com/pragdave/2012/03/be-careful-using-default_scope-and-order.html. It also plays very badly with joins.  We should get rid of them, then create explicit ordered scopes (eg. Post.recent, Harvest.recent) and use those where needed.

### LINKED ISSUE #1383: Previously planted not sorted
Plantings previously planted in this garden are unsorted. 
I think they should be most recent finished first 
screenshot_2017-08-06-13-17-07

### PR DESCRIPTION
**Pull request by Br3nda**
put the ordering in the controllers

fixes #485 
fixes #1383

### CODE REVIEW

File: app/models/harvest.rb [hunk 1]
```diff
@@ -39,7 +39,7 @@ class Harvest < ActiveRecord::Base
 
   ##
   ## Scopes
-  default_scope { joins(:owner) }
+  default_scope { joins(:owner) } # Ensures owner exists
```
Comment (Br3nda) [inline app/models/harvest.rb]: this probably should move to a mixin (or whatever those are called in rails)

**Comment by Br3nda** [comment]
I've started cleanup in controller specs to make up for removing some model specs.

File: app/views/plant_parts/index.html.haml [hunk 2]
```diff
@@ -9,7 +9,8 @@
     - if plant_part.crops.empty?
       No crops are harvested for this plant part (yet).
     - else
-      != plant_part.crops.map { |c| link_to(c, c) }.join(", ")
+      - plant_part.crops.limit(100).each do |crop|
```
Comment (CloCkWeRX) [inline app/views/plant_parts/index.html.haml]: Maybe worth sticking up into the controller the "decide how many things to show" parts? Not that we've done it here, but if we were doing pagination for this list it'd make sense there.

Comment (Br3nda) [reply app/views/plant_parts/index.html.haml]: yeah, i agree. Will move

Comment (Br3nda) [reply app/views/plant_parts/index.html.haml]: This page doesn't seem to have any links to it.... Maybe it serves no useful purpose?

File: app/controllers/accounts_controller.rb [hunk 3]
```diff
@@ -5,7 +5,7 @@ class AccountsController < ApplicationController
 
   # GET /accounts
   def index
-    @accounts = Account.all
+    @accounts = Account.all.order(created_at: :desc)
```
Comment (Br3nda) [inline app/controllers/accounts_controller.rb]: this is curiously missing pagination

Comment (CloCkWeRX) [reply app/controllers/accounts_controller.rb]: Raised #1475

**Pull request merged**